<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Peter Simonsson</author>
 <comment>This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.
</comment>
 <class>ChannelOptionsUI</class>
 <widget class="QWidget" name="ChannelOptionsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>377</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="channelTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="topicTab">
      <attribute name="title">
       <string>Topi&amp;c</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="textLabel2_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Search:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="topicHistorySearchLine">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="TopicHistoryView" name="topicHistoryView" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="TopicEdit" name="topicEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modesPage">
      <attribute name="title">
       <string>Mo&amp;des</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="4">
        <widget class="QCheckBox" name="topicModeChBox">
         <property name="text">
          <string>&amp;Topic can only be changed by channel operators</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QCheckBox" name="messageModeChBox">
         <property name="text">
          <string>&amp;No messages from outside the channel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QCheckBox" name="secretModeChBox">
         <property name="text">
          <string>&amp;Secret channel, the channel is not listed in the channel list</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="inviteModeChBox">
         <property name="text">
          <string>Only &amp;invited are allowed to join the channel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QCheckBox" name="moderatedModeChBox">
         <property name="text">
          <string>&amp;Moderated channel, only people with voice can write to the channel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="userLimitChBox">
         <property name="text">
          <string>User &amp;limit:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QSpinBox" name="userLimitEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <spacer name="spacer10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>207</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="keyModeChBox">
         <property name="text">
          <string>Channel &amp;password:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="4">
        <widget class="KLineEdit" name="keyModeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="toggleAdvancedModes">
         <property name="text">
          <string>Show &amp;Advanced Modes &gt;&gt;</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <spacer name="spacer3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>301</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QTreeView" name="otherModesList">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>112</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>&amp;Ban List</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Search:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KTreeWidgetSearchLine" name="banListSearchLine"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="banList">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Hostmask</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Set By</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time Set</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="hostmaskLabel">
           <property name="text">
            <string>Hostmask:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="hostmask"/>
         </item>
         <item>
          <widget class="QPushButton" name="updateBan">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Update Ban</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addBan">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Add Ban</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeBan">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Remove Ban</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>QLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>TopicEdit</class>
   <extends>QTextEdit</extends>
   <header>topicedit.h</header>
  </customwidget>
  <customwidget>
   <class>TopicHistoryView</class>
   <extends>QWidget</extends>
   <header>topichistoryview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>keyModeChBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>keyModeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userLimitChBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>userLimitEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
